scripts/stop_npm_proxy.ts

```diff
@@ -13,10 +13,20 @@ await execa('npm', ['config', 'set', 'registry', NPM_REGISTRY]);
 // returns the process id of the process listening on the specified port
 let pid: number;
 try {
-  const lsofResult = await execaCommand(
-    `lsof -n -t -iTCP:${VERDACCIO_PORT} -sTCP:LISTEN`
-  );
-  pid = Number.parseInt(lsofResult.stdout.toString());
+  if (process.platform === 'win32') {
+    const netStatResult = await execaCommand(
+      `netstat -n -a -o | grep ${VERDACCIO_PORT}`,
+      { shell: true }
```

Comment:
is this going to work in Powershell ?

if not then consider `shell: 'bash'`.

Reply:
`grep` command does not work in Powershell, I'll update that